docs/docs/09.2-package-management.md

```diff
@@ -0,0 +1,124 @@
+---
+id: package-management
+title: Package Management
+permalink: package-management.html
+prev: dangerously-set-inner-html.html
+next: addons.html
+---
+
+Every project uses a different system for building and deploying JavaScript. We've tried to make React as environment-agnostic as possible.  We provide React within a CDN, which makes it easy to get started.  As your application grows, you will also need a way to manage your project's code and dependencies.  We recommend using a package manager like NPM or Bower to accomplish this task.  If you've never used one of these systems, getting started can be an arduous task, but it's well-worth the investment.
+
+## CDN-hosted React
+
+We provide CDN-hosted versions of React [on our download page](/react/downloads.html). These pre-built files use the UMD module format. Dropping them in with a simple `<script>` tag will inject a `React` global into your environment. It should also work out-of-the-box in CommonJS and AMD environments.
+
+## Using master
+
+We have instructions for building from `master` [in our GitHub repository](https://github.com/facebook/react). We build a tree of CommonJS modules under `build/modules` which you can drop into any environment or packaging tool that supports CommonJS.
```

Comment:
This almost certainly shouldn't be the 2nd thing on this page. Let's move it to the bottom.

Also, referencing `build/modules` doesn't feel right. People shouldn't need to know that dir exists (and we don't really want to tell them it does) - they shouldn't be referencing anything in there. Pointing to `build/packages` for `npm install`able packages or just `build/` for browser packages is fine.